core/services/chainlink/config_general.go

```diff
@@ -89,6 +90,43 @@ func (o *GeneralConfigOpts) ParseTOML(config, secrets string) error {
 
 // New returns a coreconfig.GeneralConfig for the given options.
 func (o GeneralConfigOpts) New(lggr logger.Logger) (coreconfig.GeneralConfig, error) {
+	cfg, err := o.init()
+	if err != nil {
+		return nil, err
+	}
+	cfg.lggr = lggr
+	return cfg, nil
+}
+
+// NewAndLogger returns a coreconfig.GeneralConfig for the given options, and a logger.Logger (with close func).
+func (o GeneralConfigOpts) NewAndLogger() (coreconfig.GeneralConfig, logger.Logger, func() error, error) {
+	cfg, err := o.init()
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	// placeholder so we can call config methods to bootstrap the real logger
+	cfg.lggr, err = relaylogger.New()
```

Comment:
Temporarily log to std out while looking up the config fields - only actual relevance is to `RootDir`, which can fail on HomeDir internally, and is called indirectly from `LogFileDir`, but this solves the problem generally so we don't have to think about that.

Reply:
Maybe call it `simplelogger` or `stdoutlogger` or `bootstraplogger` or something?

Reply:
Changed to `simplelogger`. stdout and bootstrap don't seem right for the package level since it applies to the type stored in `generalConfig` too.